package.json

```diff
@@ -106,7 +106,7 @@
         "@swc/helpers": "~0.4.14",
         "antd": "5.1.5",
         "apollo-upload-client": "17.0.0",
-        "axios": "1.3.2",
+        "axios": "1.2.2",
```

Comment:
I see a downdate of the `axios` package version. What is the rational between this particular change ?